Description: Adds the dependency on JSR 305 (why is the dependency defined in the parent pom ignored?)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/surefire-api/pom.xml
+++ b/surefire-api/pom.xml
@@ -45,6 +45,12 @@
       <artifactId>jsr305</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+      <version>2.0.1</version>
+      <scope>provided</scope>
+    </dependency>
   </dependencies>
 
   <build>
